tardis/__init__.py

```diff
@@ -33,6 +33,100 @@
 logger.addHandler(console_handler)
 logging.getLogger("py.warnings").addHandler(console_handler)
 
+
+class FilterLog(object):
+    """
+    Filter Log Class for Filtering Logging Output
```

Comment:
this also seems like a very generic class? Is this from SO?

Reply:
Hello @wkerzendorf, this has been implemented taking into consideration the [Filter()](https://docs.python.org/3/library/logging.html#filter-objects) documentation present. The `filter()` function is created using [this](https://docs.python.org/3/howto/logging-cookbook.html#filters-contextual) in mind & some SO discussions for logging level specification.